apps/src/templates/AgeDialog.jsx

```diff
@@ -50,7 +47,16 @@ const styles = {
   },
 };
 
-const sessionStorageKey = 'anon_over13';
+/*
+ * SignInOrAgeDialog uses 'anon_over13' as its session storage key.
```

Comment:
I wonder if we should rename SignInOrAgeDialog to be something more specific like `AppLabIntroSigninDialog` (or something better) as it seems like we do have applab-intro specific logic in there given the importance of the different session keys.

My concern is that in the future we forget why we have these two different keys. I might also reword the text in this comment somewhat, maybe something like:
```
Though this AgeDialog has some similar behaviors to SignInOrAgeDialog, it is important that they use separate storage session keys. We want our other dialog to show up even if we've seen this dialog before, as it also contains
logic used to reset cookies and prevent PII from leaking.

Reply:
good call. i'll do this in a follow-up